### PR TITLE
Ticket 1160 vr

### PR DESCRIPTION
**Pull request by butlerpd**
Updated 4 models to correct misunderstanding of the point of VR

### CODE REVIEW

**Comment by butlerpd** [comment]
addresses #1160

File: sasmodels/models/core_shell_cylinder.py [hunk 1]
```diff
@@ -129,19 +129,12 @@ def ER(radius, thickness, length):
     """
     Returns the effective radius used in the S*P calculation
     """
+    print("radius")
```
Comment (pkienzle) [inline sasmodels/models/core_shell_cylinder.py]: Remove debug statements

File: sasmodels/models/vesicle.py [hunk 2]
```diff
@@ -2,7 +2,11 @@
 Definition
 ----------
 
-The 1D scattering intensity is calculated in the following way (Guinier, 1955)
+TThis model provides the form factor, *P(q)*, for an unilamellar vesicle. This
```
Comment (pkienzle) [inline sasmodels/models/vesicle.py]: sp of Tthis

File: sasmodels/models/vesicle.py [hunk 3]
```diff
@@ -2,7 +2,11 @@
 Definition
 ----------
 
-The 1D scattering intensity is calculated in the following way (Guinier, 1955)
+TThis model provides the form factor, *P(q)*, for an unilamellar vesicle. This 
+s model is effectively identical to the hollow sphere reparameterized to be
+more intuitive for a vesicle and normalizing the form factor by the volume of
```
Comment (pkienzle) [inline sasmodels/models/vesicle.py]: superfluous "s"

File: sasmodels/models/hollow_rectangular_prism_thin_walls.py [hunk 4]
```diff
@@ -54,10 +52,10 @@
...
   I(q) = \text{scale} \times V \times (\rho_\text{p} - \rho_\text{solvent})^2 \times P(q)
 
-where $V$ is the volume of the rectangular prism, $\rho_\text{p}$
-is the scattering length of the parallelepiped, $\rho_\text{solvent}$
-is the scattering length of the solvent, and (if the data are in absolute
-units) *scale* represents the volume fraction (which is unitless).
+where $V$ is the surface area of the rectangular prism, $\rho_\text{p}$
+is the scattering length density of the parallelepiped, $\rho_\text{solvent}$
+is the scattering length density of the solvent, and (if the data are in
+absolute units) *scale* is related to the total surface area.
```
[the remark this answers is not in the document]
Comment (butlerpd) [reply sasmodels/models/hollow_rectangular_prism_thin_walls.py]: V is clearly the surface area of the prism based on the equation in the docs.  how that translates to scale was unclear to @RichardHeenan and myself so we went with vague. If somebody can show exactly what it is, we can fix in a later release

File: sasmodels/models/hollow_rectangular_prism_thin_walls.py [hunk 5]
```diff
@@ -1,16 +1,14 @@
...
-prism with infinitely thin walls. It computes only the 1D scattering, not the 2D.
-
-
 Definition
 ----------
 
+
+This model provides the form factor, $P(q)$, for a hollow rectangular
+prism with infinitely thin walls. It computes only the 1D scattering, not the 2D.
 The 1D scattering intensity for this model is calculated according to the
```
[the remark this answers is not in the document]
Comment (butlerpd) [reply sasmodels/models/hollow_rectangular_prism_thin_walls.py]: Indeed, but it was not provided by the author so this is correct for the documentation - it does **not** currently do 2D.  When somebody does they should change the docs to reflect the change

File: sasmodels/models/hollow_rectangular_prism_thin_walls.py (hunk 4 above)
[the remark this answers is not in the document]
Comment (pkienzle) [reply sasmodels/models/hollow_rectangular_prism_thin_walls.py]: Surface area is approximately shell volume for a shell thickness of 1.

Comparing model walls with thickness=1 to thin_walls (5 pairs), it is mostly within a few percent for low q:
```
sascomp hollow_rectangular_prism,hollow_rectangular_prism_thin_walls background=0 thickness=1 -pars -sets=5
```

We can do even better if we correct for shell volume when thickness=1:
```
sascomp hollow_rectangular_prism,hollow_rectangular_prism_thin_walls background=0 -pars -double! thickness=1 -sets=5 "scale=1,(1+(8.-4*length_a*(1+b2a_ratio+c2a_ratio))/(2*length_a**2*(b2a_ratio+c2a_ratio+b2a_ratio*c2a_ratio)))" -sets=5
```
The ugly expression for scale corrects surface are for the double counted edges (without corners) and triple counted corners:
```
S = 2*(ab + ac + bc) = 2*a^2*(b/a + c/a + b/a*c/a)
E = 4*(a + b + c) = 4*a*(1 + b/a + c/a)
C = 8
scale = (S + C - E) / S  = 1 + (C - E)/S
```

I suppose this is just a curiousity, and you can leave the docs vague.

**Pull request merged**